CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v1.0.2
+### Unreleased
+#### 2023-09-06
+Added functions `publish_umm_var` and `publish_all_umm_var` to `umm_var.py`
```

Comment:
> I'm under the impression that we parse this format somewhere for release notes.

Yup - the format is parsed [here](https://github.com/nasa/earthdata-varinfo/blob/main/bin/extract-release-notes.sh). Essentially, we're splitting by lines that start with `## v`.

> I wouldn't mind a little more information about what the changes are for as opposed to just what the new functions are that are added.

Yeah - I think I agree here. As an end user, it is really good to know the new functions I can call, but knowing what they do is probably just as helpful.

> but I like to include the jira ticket that was completed as a way to track back why a change was made.

I don't have a strong feeling here. It would be including information that's mainly useful to us as developers, rather than other users who don't have visibility into Jira. The ticket references in the commit history currently serve that purpose.

Reply:
Oh I don't usually put a full link, but the ticket name DAS-1893

Reply:
I think I tend to agree with Owen on this (but I see your point!), the jira ticket ID is not really useful to an enduser who doesn't have access to jira.